schedule/yast/autoyast_create_hdd_gnome_raid_encrypted_home.yaml

```diff
@@ -0,0 +1,14 @@
+name: autoyast_create_hdd_gnome_raid_encrypted_home
```

Comment:
system is a textmode but all the yaml has gnome in the name.

Reply:
Updated to textmode.